components/FeaturesUpdate.js

```diff
@@ -47,6 +47,29 @@ const FeaturesUpdate = ({navigation}) => {
         )
       }
 
+    const performActionOnPushStatus = (status) => {
+      alert("Push permission status is - " + status)
+      console.log("Push permission status is - " + status)
+    } 
+
+    const getPushPermissionStatus = () => {
+      CustomerIO.getPushPermissionStatus((response) => {
+        performActionOnPushStatus(response)
+      })
+    }
+
+    const requestPushPermissionPrompt = () => {
+      // Todo: change
```

Comment:
Is there still anything pending or should we remove this comment now?

Reply:
Oops ! Thanks for reminding